_layouts/page.html

```diff
@@ -3,7 +3,7 @@
 ---
 <section class="hero is-medium" style="background-image: url('img/header-blur.jpg'); background-size: cover; background-position: bottom;">
   <div id="hero-head">
-    <nav class="nav">
+    <nav class="nav w3-animate-opacity w3-animate-top">
```

Comment:
These classes seem unnecessary. Please don't add unnecessary classes when you can easily animate via standard element selectors.